tokio/src/sync/mpsc/unbounded.rs

```diff
@@ -245,4 +245,25 @@ impl<T> UnboundedSender<T> {
     pub async fn closed(&mut self) {
         self.chan.closed().await
     }
+    /// Checks if the channel has been closed. This happens when the
+    /// [`UnboundedReceiver`] is dropped, or when
+    /// the [`UnboundedReceiver::close`] method is called.
```

Comment:
Can you reflow text to 80 characters?

Reply:
Ok, I wrapped "the" to the previous line.
But [`UnboundedReceiver::close`] is soooo long, and it can't fit previous line.